app/components/DocumentBreadcrumb.tsx

```diff
@@ -18,6 +18,8 @@ type Props = {
   children?: React.ReactNode;
   document: Document;
   onlyText?: boolean;
+  reverse?: boolean;
+  reversedLength?: number;
```

Comment:
It doesn't make sense for this to apply only to reversing, a prop is fine but it should be called `maxLength` or `maxDepth` and apply to both cases